### PR TITLE
Completed JS Functions Practice

### PR DESCRIPTION
**Pull request by macyhongg**
image
image
image

### CODE REVIEW

File: javascript-functions-practice/get-area-of-circle.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* exported getAreaOfCircle */
+
+function getAreaOfCircle(radius){
+  let RadiusSquared = radius * radius;
```
Comment (BrettAlbright) [inline javascript-functions-practice/get-area-of-circle.js:4]: this variable should be lowercase

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
sorry, hit the wrong button

**Comment by macyhongg** [comment]
should I add and commit again to get rid of this?
image

**Pull request merged**